cmd/genji/main.go

```diff
@@ -26,38 +25,91 @@ func main() {
 		},
 	}
 
+	app.Commands = []*cli.Command{
+		{
+			Name:      "insert",
+			Usage:     "Allows inserting documents from the command line",
+			UsageText: "genji insert [options] [arguments...]",
+			HideHelp:  true,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     "engine",
+					Aliases:  []string{"e"},
+					Usage:    "Store engine bolt or badger",
```

Comment:
```suggestion
					Usage:    "name of the engine to use, options are 'bolt' or 'badger'. defaults to bolt",
```

Reply:
You should use the `Value` field to setup the default value: https://godoc.org/github.com/urfave/cli#StringFlag